src/AppInstallerRepositoryCore/Microsoft/Schema/1_1/Interface.h

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#pragma once
+#include "Microsoft/Schema/ISQLiteIndex.h"
+#include "Microsoft/Schema/1_0/Interface.h"
+
+
+namespace AppInstaller::Repository::Microsoft::Schema::V1_1
+{
+    // Interface to this schema version exposed through ISQLiteIndex.
+    struct Interface : public V1_0::Interface
+    {
+        // Version 1.0

```

Comment:
>1.0 [](start = 19, length = 3)

1.1?

Reply:
No, this 1.0 refers to the interface version where the functions come from.  These are 1.0 functions.  If 1.1 added new accessor functions, they would be in a 1.1 section.